src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').reduce((acc, declaration) => {
+    const [property, value] = declaration.split(':').map((part) => part.trim());
+
+    if (property && value) {
+      acc[property] = value;
+    }
+
+    return acc;
+  }, {});
```

Comment:
The implementation of the `convertToObject` function is mostly correct. However, consider using more descriptive variable names than `acc` for the accumulator in the `reduce` method to improve code readability.